Templates/StreamDeck.PluginTemplate.Csharp/content/RegisterPluginAndStartStreamDeck.sh

```diff
@@ -16,19 +14,25 @@ uuid=$(sed -n 's/.*"UUID": "\(.*\)"/\1/p' manifest.json)
 
 pluginName=$(echo "${uuid}" | tr -d \" | sed -e "s/${SUFFIX}$//")
 
+#Define Plugin Directory
+PLUGIN_DIR="${HOME}/Library/Application Support/com.elgato.StreamDeck/Plugins/${pluginName}.sdPlugin"
```

Comment:
Have you verified setting this up as a variable works? I don't have time to grab the code quite yet to test it out. When I'd set it up as a var it continually blew up on me. "if you say so..." :)

Reply:
@bradygaster I think this might have been caused by the .editorconfig forcing `crlf` on all files. It was causing crazy problems for me on Sunday.

Reply:
it was a combination of using ~/ for path expansion and having the "\" escape character in "Application Support".